scripts/app-stream/SETUP.md

```diff
@@ -29,7 +29,7 @@ Note: Please set up your [AWS Profile](https://docs.aws.amazon.com/cli/latest/us
 cd ~\Documents
 
 # Pull the Image Builder script from Github
-Invoke-WebRequest -Uri https://raw.githubusercontent.com/awslabs/service-workbench-on-aws/feat-secure-workspace-egress/scripts/app-stream/buildImage.ps1 -OutFile buildImage.ps1
+Invoke-WebRequest -Uri https://raw.githubusercontent.com/awslabs/service-workbench-on-aws/feat-appstream-ui/scripts/app-stream/buildImage.ps1 -OutFile buildImage.ps1
```

Comment:
I think this branch needs to be mainline?

Reply:
Eventually, yes. I can change it now too.

Reply:
Either keep it `feat-secure-workspace-egress` or change to `mainline`. My preference is change to mainline now since we might forget it later.